static/customview.js

```diff
@@ -110,6 +121,9 @@ function updateMCHCombinedBoxPlot(mmu_gid, hsa_gid) {
 function updateGeneElements() {
     var geneSelected = $('#geneName option:selected').val();
     if (geneSelected != 'Select..') {
+        if (typeof(Storage) !== 'undefined') {
+            localStorage.setItem(species, $('#geneName option:selected').text());
```

Comment:
Howard - It would be nice to automatically display the ortholog of the current gene when switching species. For example, if the user is currently viewing Rorb in mmu, then they click hsa, the gene selection should default to RORB (ortholog of Rorb). Is this possible?